ColorPatterns/Scenes/StartMenu/StartMenuController.swift

```diff
@@ -38,13 +41,14 @@ final class StartMenuController: UIViewController {
 private extension StartMenuController {
 
     private func setupDifficultyButtonColor(button: Int) {
+        // TODO: crash if I do:  setupDifficultyButtonColor(button: 52) - this should be forbidden
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/6055d8cf2ff9610160e4bc6d457c28b6db6bd01c/ColorPatterns/Scenes/StartMenu/StartMenuController.swift#L44"></span>TODOs should be resolved (crash if I do:  setupDifficult…).<br />
<code>todo</code> <code>StartMenuController.swift:44</code></td>
    </tr>
  </tbody>
</table>